src/manage.py

```diff
@@ -3,6 +3,10 @@
 import os
 import sys
 
+from dotenv import load_dotenv
+
+load_dotenv()
```

Comment:
please pass `override=True` to make sure that `os.environ.get` will return same data as we have in `.env`.
Or refactor settings module to use dotenv library.

Reply:
Fixed